src/CLI/clitree/cli-xml/image.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE CLISH_MODULE [
+]>
+<CLISH_MODULE
+      xmlns="http://www.dellemc.com/sonic/XMLSchema"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:xi="http://www.w3.org/2001/XInclude"
+      xsi:schemaLocation="http://www.dellemc.com/sonic/XMLSchema 
+      http://www.dellemc.com/sonic/XMLSchema/clish.xsd"
+      >
+  <VIEW name="enable-view">
+    <COMMAND
+         name="show image-list"
+         help="Show list of images"
+         >
+         <ACTION>
+              python $SONIC_CLI_ROOT/sonic-cli-img.py get_openconfig_platform_components image_show.j2 ${__full_line}
+         </ACTION>
+    </COMMAND>
+
+    <COMMAND
+         name="image"
+         help="image related commands"
+         >
+         <ACTION>
+              python $SONIC_CLI_ROOT/sonic-cli-img.py get_openconfig_platform_components image_show.j2 ${__full_line}
+         </ACTION>
+    </COMMAND> 
+    <COMMAND
+	 name="image set-default"
+	 help="set default image">
+         <PARAM
+	      name="img-name"
+	      help="image name"
+	      ptype="STRING"
+	      >
+	 </PARAM>
+         <ACTION>
+	      python $SONIC_CLI_ROOT/sonic-cli-img.py get_openconfig_platform_components ${img-name} ${__full_line}
+	 </ACTION>
+    </COMMAND>
+    <COMMAND
+	 name="image install"
+	 help="install image">
+         <PARAM
+	      name="img-path"
```

Comment:
Do we need to have separate commands for URL or http image install? If we can have separate commands, we could specify HTTP_TYPE as ptype for http install and it does regex matching. @joyas-joseph Could you give your suggestion?

Reply:
Could you confirm and work with @joyas-joseph on this?